keyboards/momokai/kei_6/keymaps/via/keymap.c

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+                KC_MUTE,
+        KC_Z,   KC_X,   KC_C,
+        KC_GRV,   KC_ESC,   KC_F2
+    ),
+    [1] = LAYOUT(
+                KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS
+    ),
+    [2] = LAYOUT(
+                KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS
+    ),
+    [3] = LAYOUT(
+                KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS
+    )
```

Comment:
This isn't needed.  Any unpopulated layers are automatically filled in with transparency. 
```suggestion
```